### DISCUSSION TITLE
State of InnerSource 翻訳

### DISCUSSION THREAD

**Discussion by yuhattor** [InnerSourceCommons/jp-roadmap#20] [Announcements]
State of InnerSource というレポートがあります。

- [サマリースライド](https://www.youtube.com/watch?v=pKV6RH3p8XU) (Google Slides) の翻訳
- [レポート自体](https://innersourcecommons.org/learn/research/state-of-innersource-survey-2021/) (Google Docs) の翻訳

こういったレポートを日本語に翻訳することは意味があることだと思います

**Comment by masskaneko** [comment]
先月見ました。コミュニティコールでも解説されていましたね。
いまの状況を伝えるのに役立つと思います。
年1で量も多くはないのでそんなに負担ではないと思いますし。
勉強会のネタにもなると思います。
